app/src/main/java/com/cornellappdev/android/eateryblue/ui/components/login/AccountPage.kt

```diff
@@ -0,0 +1,562 @@
+                items(
+                    loginViewModel.getTransactionsOfType(
+                        accountState.accountFilter,
+                        filterText
+                    )
+                ) { it ->
+                    val inputFormatter =
+                        DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSX")
+                    val outputFormatter = DateTimeFormatter.ofPattern("h:mm a · EEEE, MMMM d")
+                    val dateTime = LocalDateTime.parse(it.date, inputFormatter)
+                    Row(
+                        modifier = Modifier
+                            .height(64.dp)
+                            .padding(horizontal = 16.dp),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Column(modifier = Modifier.weight(1f)) {
+                            Text(text = "${it.location}", style = EateryBlueTypography.button)
+                            Text(
+                                text = outputFormatter.format(dateTime),
+                                style = EateryBlueTypography.subtitle2
+                            )
+                        }
+                        var amtColor by remember { mutableStateOf(Color.Unspecified) }
+                        var amtString by remember { mutableStateOf("$0.00") }
+                        when {
+                            it.transactionType == 3 -> {
+                                amtString = "+$%.2f".format(it.amount)
+                                amtColor =
+                                    Color(LocalContext.current.resources.getColor(R.color.green))
+                            }
+
+                            it.amount?.toInt() == 0 -> {
+                                amtString = "$0.00"
+                                amtColor = Color.Black
+                            }
+
+                            else -> {
+                                amtString = "-$%.2f".format(it.amount)
+                                amtColor =
+                                    Color(LocalContext.current.resources.getColor(R.color.red))
+                            }
+                        }
+                        Text(
+                            text = amtString,
+                            modifier = Modifier.weight(0.2f),
+                            color = amtColor,
+                            textAlign = TextAlign.Right,
+                            style = EateryBlueTypography.button,
+                        )
+
+                    }
+                    Spacer(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .height(1.dp)
+                            .background(GrayZero, CircleShape)
+                    )
+                }
+            }
+        }
+
+    }
+}
+
+
+@Composable
+fun MealPlanRow(accountName: String, accountType: AccountType, loginViewModel: LoginViewModel) {
```

Comment:
Great component! Just rename to something `AccountBalanceRow` since it's being used for things besides Meal Swipes